Clicker/ViewControllers/PollBuilderViewController.swift

```diff
@@ -286,7 +283,9 @@ class PollBuilderViewController: UIViewController, QuestionDelegate, PollBuilder
             let question = frPollBuilder.questionTextField.text ?? ""
             delegate.startPoll(text: question, type: .freeResponse, options: [], state: .live)
         }
-        
+        guard let nav = self.presentingViewController as? UINavigationController else { return }
+        guard let cardController = nav.topViewController as? CardController else { return }
+        cardController.navigationController?.setNavigationBarHidden(false, animated: true)
```

Comment:
Since `CardController` already conforms to `PollBuilderViewControllerDelegate`, can we just add a delegate function for cardController to show the nav bar instead of doing this unwrapping here?